doc/usage/harbor.md

```diff
@@ -0,0 +1,159 @@
+> is also automatically created in the openstack Swift object store.
+
+> Warning: Swift container and ec2 credentials are not removed when the workload cluster is deleted.
+
+## Default deployment
+
+By default, Harbor will be deployed with these options:
+```terraform
+harbor_config = {
+  domain_name: "",
+  issuer_email: "",
+  persistence: false,
+  database_size: "1Gi",
+  redis_size: "1Gi",
+  trivy_size: "5Gi"
+}
+```
+
+These options are templated to the mgmt server and used as environment variables
+during the creation of the workload cluster:
+```bash
+$ cat ~/cluster-defaults/harbor-settings
+DEPLOY_HARBOR=true
+HARBOR_DATABASE_SIZE=1Gi
+HARBOR_DOMAIN_NAME=
+HARBOR_ISSUER_EMAIL=
+HARBOR_PERSISTENCE=false
+HARBOR_REDIS_SIZE=1Gi
+HARBOR_TRIVY_SIZE=5G
+```
+
+When the Harbor is deployed, you can check the status, e.g. by:
+```bash
+$ flux get helmrelease -n default
+NAME    REVISION        SUSPENDED       READY   MESSAGE                          
+harbor  1.12.3          False           True    Release reconciliation succeeded
+```
+or you can simply check if pods are running by `kubectl get pods`.
+
+Harbor components are deployed as deployments and statefulsets:
+```bash
+$ kubectl get deploy,sts
+NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/harbor-core         2/2     2            2           11m
+deployment.apps/harbor-exporter     1/1     1            1           11m
+deployment.apps/harbor-jobservice   2/2     2            2           11m
+deployment.apps/harbor-nginx        1/1     1            1           11m
+deployment.apps/harbor-portal       2/2     2            2           11m
+deployment.apps/harbor-registry     2/2     2            2           11m
+
+NAME                               READY   AGE
+statefulset.apps/harbor-database   1/1     11m
+statefulset.apps/harbor-redis      1/1     11m
+statefulset.apps/harbor-trivy      2/2     11m
+```
+
+Default options deploy Harbor as clusterIP service without pvc persistence for database, redis and trivy.
+See `terraform/files/kubernetes-manifests.d/harbor/envs/clusterIP/` for further details.
+You can access it via the k8s service `harbor`, for example, *port-forward* it and access it at http://localhost:8080:
+```bash
+$ kubectl get svc harbor -o wide
+NAME     TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE   SELECTOR
+harbor   ClusterIP   10.109.57.148   <none>        80/TCP    11m   app=harbor,component=nginx,release=harbor
+$ kubectl port-forward svc/harbor 8080:80
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+```
+
+Admin username is `admin` and password can be obtained from the secret by:
+```bash
+kubectl get secret harbor-secrets -o jsonpath='{.data.values\.yaml}' | base64 -d | yq .harborAdminPassword
+```
+
+## Configuration options
+
+### Persistence
+
+By default, `persistence` is set to `false`.
+When persistence is set to `true`, e.g. `harbor_config = {persistence: true}`,
```

Comment:
```suggestion
When persistence is set to `true`, i.e. `harbor_config = {persistence: true}`,
```

Reply:
Addressed in 4b7e7455f2da5ebec9f7ca75d7a0bc1bbec65174